src/main/kotlin/ua/anastasiia/finesapp/controller/FineController.kt

```diff
@@ -1,34 +1,96 @@
 package ua.anastasiia.finesapp.controller
 
+import jakarta.validation.Valid
+import org.bson.types.ObjectId
+import org.springframework.web.bind.annotation.DeleteMapping
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.PatchMapping
 import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.PutMapping
 import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
 import org.springframework.web.bind.annotation.RestController
+import ua.anastasiia.finesapp.dto.CarRequest
 import ua.anastasiia.finesapp.dto.FineRequest
-import ua.anastasiia.finesapp.dto.FineResponse
-import ua.anastasiia.finesapp.entity.Fine
-import ua.anastasiia.finesapp.service.FineService
+import ua.anastasiia.finesapp.dto.TotalFineSumResponse
+import ua.anastasiia.finesapp.dto.TrafficTicketRequest
+import ua.anastasiia.finesapp.entity.MongoFine
+import ua.anastasiia.finesapp.service.MongoFineService
+import java.time.LocalDate
 
 @RestController
 @RequestMapping(value = ["/fines"])
-class FineController(val fineService: FineService) {
+@Suppress("TooManyFunctions")
+class FineController(val mongoFineService: MongoFineService) {
```

Comment:
```suggestion
class FineController(val fineService: FineService) {
```
service is responsible for business logic, it should not depend on database details